src/cclib/io/moldenwriter.py

```diff
@@ -24,6 +25,11 @@ def _title(self, path):
     def _coords_from_ccdata(self, index):
         """Create [Atoms] section using geometry at the given index."""
 
+        if not (hasattr(self.ccdata, 'atomcoords') and
+                hasattr(self.ccdata, 'atomnos') and
+                hasattr(self.ccdata, 'natom')):
+                raise MissingAttributeError('Could not parse necessary outputs to write molden file.')
+
```

Comment:
Made the change here https://github.com/cclib/cclib/pull/388/files#diff-599bcb2b83c5ef046682dfaa37fd74e1R20.